tests/integration/list.bats

```diff
@@ -38,6 +38,12 @@ function teardown() {
   [[ "${lines[2]}" == *"test_box2"*[0-9]*"running"*$BUSYBOX_BUNDLE*[0-9]* ]]
   [[ "${lines[3]}" == *"test_box3"*[0-9]*"running"*$BUSYBOX_BUNDLE*[0-9]* ]]
 
+  ROOT=$HELLO_BUNDLE runc list -q
+  [ "$status" -eq 0 ]
+  [[ "${lines[0]}" == *"test_box1"* ]]
```

Comment:
Thanks for your carefully review. It is my mistakes. It should be `[[ "${lines[0]}" == "test_box1" ]]` to match lines only contain container-id.

Fixed done.